tests/Storage.test.ts

```diff
@@ -245,21 +245,21 @@ describe ('Test for storing block data in the database', () =>
     it ('Error-handling test when writing a block.', async () =>
     {
         let sample_data0 = JSON.parse(sample_data_raw[0].replace(/([\[:])?(\d+)([,\}\]])/g, "$1\"$2\"$3"));
-        await ledger_storage.putBlocks(sample_data0);
-        await assert.rejects(ledger_storage.putBlocks(sample_data0),
+        let block = Block.fromJSON(sample_data0);
```

Comment:
I think it's missing. ;)

Reply:
Oops, I actually caught a third instance :D